bin/create-user.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env node, browser */
```

Comment:
Instead of making a `bin/` directory, what about naming it `scripts/`? I think it's more descriptive, and none of these are actual binaries.

Reply:
`bin` is the name of the existing directory. Renaming it is out of scope, I think.